cloud-aws/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/component/AwsLaunchTest.java

```diff
@@ -156,6 +157,9 @@
     @MockBean
     private CustomAmazonWaiterProvider customAmazonWaiterProvider;
 
+    @MockBean
```

Comment:
this is a mock bean here but there is no assertion or verification on this

Reply:
Changed things up to mock in a way that makes more sense with what's going on and the original error.